shared/src/main/java/gov/hhs/cdc/trustedintermediary/metadata/MetaDataEntry.java

```diff
@@ -0,0 +1,17 @@
+package gov.hhs.cdc.trustedintermediary.metadata;
+
+import java.util.Date;
+
+/** An instance of a metadata event to be used for internal troubleshooting of messages */
+public class MetaDataEntry {
+
+    public MetaDataStep entryStep;
+    public Date entryTime;
```

Comment:
I would make this an `Instant`.  `Date` is the old Java time construct and we should use the new time constructs.

Reply:
done.